### ISSUE TITLE
How can i do an Broadcast Server instead an Echo Server?

### ISSUE THREAD

**Issue by mok-liee** [vtortola/WebSocketListener#57]
Hi, is it possible to make a Method to send a Broadcast Message to all connected Clients?

Greetings mok

**Comment by vtortola** [comment]
Yes, you will need to implement your own presence control. If you are planning to run your application in only one server you can have a concurrent collection where you add clients on connection and remove them on disconnection. If you plan to use multiple servers, probably is better idea to use a service bus like RabbitMQ.
